src/ratings/components/reviewComponents/Review.jsx

```diff
@@ -10,7 +10,7 @@ const Review = (props) => {
     <div style={{margin: '30px'}}>
       <div style={{clear: 'both'}}>
         <div style={{float: 'left'}}>
-          <StarCount id={props.review.review_id} starNumber={props.review.rating}/>
+          <StarCount key={Math.random()} id={props.review.review_id} starNumber={props.review.rating}/>
```

Comment:
Key could be equal to review_id since that's supposed to be a  unique value and best practice

Reply:
Got it, I'll make those adjustments. The Selected Style is not my component though. Thanks for feedback.